modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -1075,6 +1075,8 @@ CV__DNN_INLINE_NS_BEGIN
      protected:
          struct Impl;
          Ptr<Impl> impl;
+
+         void initInputSize();
```

Comment:
Non public methods should go into `Impl` (PImpl idiom).

Reply:
Will fix it soon